src/components/Layout/Footer.js

```diff
@@ -3,7 +3,7 @@ import React from 'react'
 const Footer = () => (
   <div className="footer">
     <hr/>
-    <p style={{ color: 'grey', textAlign: 'left' }}>Developed by Isaac A., Will A., Steven D., Jason F.</p>
+    <p style={{ color: 'grey', textAlign: 'right' }}>Developed by Isaac A., Will A., Steven D., Jason F.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
```

Comment:
Maybe we can add some padding instead of non-breaking space?

Reply:
Sure! Feel free to make any changes. 

I was searching for a solution and that was the first one that came up.

I can look at it later on this weekend too if you don't get around to it.